backend/metadata/databases/default/tables/public_Attendence.yaml

```diff
@@ -2,19 +2,9 @@ table:
   name: Attendence
   schema: public
 object_relationships:
-- name: Person
-  using:
-    foreign_key_constraint_on: PersonId
 - name: Session
   using:
     foreign_key_constraint_on: SessionId
-select_permissions:
-- permission:
-    columns:
-    - Attending
-    - Id
-    filter:
-      Person:
-        AuthId:
-          _eq: X-Hasura-User-Id
-  role: user
```

Comment:
Why did you removed these permissions? We need to fetch all Attendance Records associated with the current user to display the Attendance section on the course page.

Reply:
Reintroduced this.